fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/RedisPermissionsRepository.java

```diff
@@ -103,33 +101,31 @@ public RedisPermissionsRepository put(@NonNull UserPermission permission) {
           }
         });
 
-    try (Jedis jedis = jedisSource.getJedis()) {
-      Pipeline deleteOldValuesPipeline = jedis.pipelined();
-      Pipeline insertNewValuesPipeline = jedis.pipelined();
+    try {
+      redisClientDelegate.withMultiKeyPipeline(pipeline -> {
```

Comment:
@robzienert -- does my respective use of the `redisClientDelegate` in here look good?